src/jpegxl.imageio/jxloutput.cpp

```diff
@@ -263,7 +263,8 @@ JxlOutput::save_image()
                                      size);
     DBG std::cout << "status = " << status << "\n";
     if (status != JXL_ENC_SUCCESS) {
-        DBG std::cout << "JxlEncoderAddImageFrame failed.\n";
+        JxlEncoderError err = JxlEncoderGetError(m_encoder.get());
+        DBG std::cout << "JxlEncoderAddImageFrame failed with error " << err << "\n";
```

Comment:
Don't forget a call to errorfmt() here.

Reply:
Thanks. I guess 4 channel image, i.e. RGB+α is not properly handled yet. Will fix ASAP and try w/ Blender.